settings.py

```diff
@@ -116,3 +112,11 @@
 }
 
 WATERMARK_IMAGE = None
+
+#: celery beat config
+CELERY_BEAT_SCHEDULE = {
```

Comment:
imo it would be better to include it from default settings:
```py
from newsroom.default_settings import CELERY_BEAT_SCHEDULE as CELERY_BEAT_SCHEDULE_DEFAULT

CELERY_BEAT_SCHEDULE = {key: val for key, val in CELERY_BEAT_SCHEDULE_DEFAULT if key == 'newsroom.company...'}
```